nemo/collections/asr/models/label_models.py

```diff
@@ -87,7 +87,18 @@ def list_available_models(cls) -> List[PretrainedModelInfo]:
 
     def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         self.world_size = 1
-        self.cal_labels_occurrence = False
+        self.cal_labels_occurrence_train = False
+        self.labels_occurrence = None
+
+        if 'loss' in cfg:
```

Comment:
You may assume loss in cfg but check if 'weight'. in params and if its there check for auto. Also check if pretrained models working or not